### ISSUE TITLE
Which exact WSIs are selected for EGFR prediction?

### ISSUE THREAD

**Issue by monajalal** [ncoudray/DeepPATH#101]
Hi Nicolas,

I am interested to reproduce your results however, from the paper, it was not clear to me, which exact WSIs you have picked for train, validation, and test set. Could you please provide me with that information either here or via jalal@bu.edu ?

Thanks a lot,
Mona Jalal

**Comment by ncoudray** [comment]
Hi Mona,

You can follow the example in:
 https://github.com/ncoudray/DeepPATH/tree/master/DeepPATH_code/example_TCGA_lung
For mutations, we now favor binary classifications which are easier to deal with. Example at the end of this thread: #37 
Best,
Nicolas

**Comment by monajalal** [comment]
Hi Nicolas 

Thanks for your response. Could you point me to the exact file that shows which WSIs from LUAD are selected for EGFR and non-EGFR in your experiments? Looking at the link that you shared above, I cannot find anything labeled as "EGFR"

![Screen Shot 2022-03-18 at 10 08 34 PM](https://user-images.githubusercontent.com/1892917/159102842-d712c611-2ec5-4423-af81-4e2a843a7c4f.png)

**Comment by monajalal** [comment]
@ncoudray is this the file you used?

https://github.com/ncoudray/DeepPATH/blob/3be13f2919186bcada82a7c6e238de16bcc8196c/DeepPATH_code/example_TCGA_lung/labels_r3.txt

I already know that in my data 88 WSI from 566 LUAD is EGFR. However, I do not know which exact WSIs from LUAD you have used for fine-tuning for EGFR prediction. Would you please be able to tell that? It seems you have not used all of the non-EGFR WSIs from the LUAD.

**Comment by ncoudray** [comment]
Hi - yes this is the list. All those which are not marked as "EGFR" are considered as non-EGFR in this list.
Best,
N.
